website/index.html

```diff
@@ -110,11 +112,5 @@ <h2>
          <footer>
             <p>Footer notes | Footer notes</p>
         </footer>
-        <div class="gradient-contatiner">
-            <div class="gradient">
-            </div>
-            <div class="gradient-push">
-            </div>
-        </div>
     </body>
```

Comment:
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/16405438/183031318-7fbe0a6c-8a34-416a-9e1b-f598005fa036.png)

Reply:
use this structure. and make width of fixed element the width of the border dash and delete z-indexes

Reply:
1f1255e

Reply:
consider div on z-index, can you do it more precise?

Reply:
![image](https://user-images.githubusercontent.com/16405438/185451332-4552e3f9-2760-4544-b2f0-672dda6053ed.png)
Problem with background

Reply:
> dont think this is possible. happy to see a solution tho

First off you could start by decrementing every z-index by 1. The footer and header will then be at 1 and the line would be at 0, which is default, so no need to set it.

Reply:
I believe the proper way to do this is with svg masking. You can just do this in second iteration if you want

Reply:
we find a correct solution in another PR